src/query/query_parser/query_grammar.rs

```diff
@@ -115,6 +121,7 @@ mod test {
         test_parse_query_to_ast_helper("abc:a b", "(abc:\"a\" \"b\")");
         test_parse_query_to_ast_helper("abc:\"a b\"", "abc:\"a b\"");
         test_parse_query_to_ast_helper("foo:[1 TO 5]", "foo:[\"1\" TO \"5\"]");
+        test_parse_query_to_ast_helper("[1 TO 5]", "[\"1\" TO \"5\"]");
         test_parse_query_to_ast_helper("foo:{a TO z}", "foo:{\"a\" TO \"z\"}");
```

Comment:
Wow... why the hell did Lucene decide to use curly brackets for that O_o

Reply:
Parens are already taken. Backwards braces probably cause parsing ambiguities. Better options?